src/goto-programs/goto_coverage.cpp

```diff
@@ -192,11 +233,64 @@ void goto_coveraget::gen_cond_cov()
           guard = handle_single_guard(guard);
 
           exprt pre_cond = nil_exprt();
+          pre_cond.location() = it->location;
           gen_cond_cov_assert(guard, pre_cond, goto_program, it);
         }
+
+        else if (it->is_assign())
```

Comment:
Please document the body of the if-statements.

Reply:
Will do